bot.py

```diff
@@ -179,6 +181,16 @@ def _setup_api(self):
              print('Exception during print player profile')
         self.update_inventory();
 
+    def _setup_tree(self):
+        tasks = []
+
+        for taskConfig in self.config.tasks:
+            taskRef = globals()[taskConfig['type']]
```

Comment:
I'm not very familiar with python, the thinking here is that we want to be able to instantiate these tasks by name. It's probably the right choice to just put a switch statement in for now. If we were trying to support a plugin system at this point, we would need to figure out some way to dynamically pull those in and instantiate by string name. Overengineering at this point though. :)